multinet/swagger/graph_nodes.yaml

```diff
@@ -0,0 +1,21 @@
+Retrieve the nodes of a graph
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/graph"
+  - $ref: "#/parameters/offset"
+  - $ref: "#/parameters/limit"
+
+responses:
+  200:
+    description: A list of nodes from the requested graph
+    schema:
+      $ref: "#/definitions/entity_names"
+
```

Comment:
Can we do a richer example here?

Reply:
Oops, I think this is a mistake actually. I removed `entity_names` because it's not markedly different from `entity_name` embedded in a list type to justify it as a separate model. Will fix (and will look at enriching the example value if possible).

Reply:
66e11acd1cff83b31a8fe5d1dd4315f7c525a0f7